ts/features/bonus/bpd/saga/orchestration/optInPaymentMethods/optInShouldShowChoiceHandler.ts

```diff
@@ -33,13 +39,19 @@ export function* optInShouldShowChoiceHandler(): Generator<
   any
 > {
   // Load the information about the participation of the user to the bpd program
-  yield* put(bpdAllData.request());
+  yield* put(bpdLoadActivationStatus.request());
   const bpdAllDataResponse = yield* take<
```

Comment:
Maybe this variable name should be changed?

Reply:
fixed in [c0a7f02](https://github.com/pagopa/io-app/pull/4173/commits/c0a7f026ddaf56b69670b3d3b971c867f9d05bee)